src/iks/iks.py

```diff
@@ -157,28 +163,29 @@ def iks(args):
         resource_group_id = UserInfo.get_resource_id()
         user_ACL = AclRuleCreator(resource_group_id, UserInfo.vpc_name, UserInfo.cis_api_key)
         user_ACL.check_network_acl()
-        '''
-        # 2. Generate certificate in manager if necessary
         
-        UserInfo.cert_name="cis-cert"
-        '''
-        cms_id = UserInfo.get_cms()
-        # print("\n"+cms_id)
-        user_cert = SecretCertificateCreator(
-            cis_crn=UserInfo.crn,
-            cluster_id=UserInfo.iks_cluster_id,
-            cis_domain=UserInfo.cis_domain,
-            cert_manager_crn=cms_id,
-
-            token=UserInfo.token["access_token"],
-            cert_name=UserInfo.cert_name
-            )
-        user_cert.create_secret()
+        # 2. Generate certificate in manager if necessary
+        print("Can order a new TLS Certificate and import as a secret to IKS or use the default secret already in IKS")
```

Comment:
Maybe a clearer message here saying we're currently using the default IKS secret but if they wish we can order a new TLS certificate and import that for the secret.